financial-connections/src/main/java/com/stripe/android/financialconnections/domain/NativeAuthFlowRouter.kt

```diff
@@ -15,21 +15,21 @@ internal class NativeAuthFlowRouter @Inject constructor(
     private val debugConfiguration: DebugConfiguration
 ) {
 
-    fun nativeAuthFlowEnabled(sync: SynchronizeSessionResponse): Boolean {
+    fun nativeAuthFlowEnabled(manifest: FinancialConnectionsSessionManifest): Boolean {
```

Comment:
Minor change - we don't need to pass the entire sync response to the `Router`. We can just pass the manifest.